packages/mux-player/src/html.ts

```diff
@@ -149,67 +150,22 @@ export class TemplateResult {
   }
 }
 
-const stringsCache = new Map();
-const defaultProcessor = createProcessor(processPart);
-export function html(strings: TemplateStringsArray, ...values: unknown[]): TemplateResult {
-  const staticStrings: any = [''];
-  const dynamicValues: any[] = [];
-  let staticValues;
-  let hasStatics = false;
-
-  // Here the unsafe static values are moved from the string expressions
-  // to the static strings so they can be used in the cache key and later
-  // be used to generate the HTML via the <template> element.
-  const join = (strs: TemplateStringsArray, vals: any[] = []) => {
-    staticStrings[staticStrings.length - 1] = staticStrings[staticStrings.length - 1] + strs[0];
-
-    vals.forEach((dynamicValue, i) => {
-      if ((staticValues = dynamicValue?.$static$) !== undefined) {
-        staticValues.forEach((staticValue: TemplateResult) => {
-          join(staticValue.strings, staticValue.values);
-        });
-
-        staticStrings[staticStrings.length - 1] = staticStrings[staticStrings.length - 1] + strs[i + 1];
-        hasStatics = true;
-      } else {
-        dynamicValues.push(dynamicValue);
-        staticStrings.push(strs[i + 1]);
+const defaultProcessor = {
```

Comment:
question(blocking): am I missing something or did we loose our `stringsCache` and any equivalent? Could this contribute to issues we've been discussing re:rendered template GC & duplication?

Reply:
this logic was to add unsafe statics like a dynamic tag name but this isn't needed anymore because custom themes will be identified by an id attribute. not custom element tags anymore.

I don't think it will have any effect on the issue but lets see. it can only be positive.

Reply:
I'm not sure if I agree with "it can only be positive," if one of our concerns is GC of the `WeakMap` references (I thought you said the `stringsCache` was the thing guaranteeing a maintained reference), but we can move forward with this and find out.